bpm_check.js

```diff
@@ -0,0 +1,8 @@
+function IsBPMUnderControl(bpm){
```

Comment:
This is very similar to IsRespirationOk. In JavaScript you can make a function to return a function. So you could have a function that returns different functions to check against different limits, so you can have one function per limit without having to actually write it. Try it, will be interesting!